requirements.txt

```diff
@@ -3,3 +3,4 @@ coverage
 mock
 unidecode
 backports.shutil_get_terminal_size; python_version < '3.3'
+lxml
```

Comment:
Just committed that.

Sorry for breaking the build. I just realize that I could actually ask Travis to build before submitting my pull request. As a side node, in some project, I have been able to get rid off the `requirements.txt` and to place everything in the `setup.py`, but I guess only more recent version of PIP would support that.

Reply:
When we drop Python 2 support, it would probably be a good time to look into simplifying the packaging as much as possible.  We'll just leave the craziness for now. :)